### DISCUSSION TITLE
How to know the development mode?

### DISCUSSION THREAD

**Discussion by dninomiya** [facebook/docusaurus#7108] [Q&A] [answered]
This information is needed because some information is only visible on the development server.

Thank you for your support.

**Comment by dninomiya** [comment, marked as answer]
Resolved with custom fields and dotenv

**Reply by Josh-Cena** [reply]
@dninomiya Sorry for the late, but you can know this through `process.env.NODE_ENV`. See https://docusaurus.io/docs/advanced/ssg#node-env

**Reply by dninomiya** [reply]
Thank you for the useful information!
